picard/ui/scripteditor.py

```diff
@@ -345,11 +352,6 @@ def make_menu(self):
         """
         config = get_config()
         main_menu = QtWidgets.QMenuBar()
-        base_font = self.ui.file_naming_editor_close.font()
-        main_menu.setStyleSheet(
-            "QMenuBar { font-family: %s; font-size: %upt; }" %
-            (base_font.family(), base_font.pointSize())
-        )
```

Comment:
@rdswift I could not figure out what this was supposed to fix. Do we still need this?

Reply:
I had to include that here because Qt was using a different font face and size for the menu bar here on my Windows system.  It seems that the menu was not inheriting the style properly.  I must have done something wrong, but wasn't sure how to fix it, so that was a quick hack.

Reply:
Ok. At least for me the menu shows up with exactly the same font and font size as on the main window.

Was this on Windows 10 or 7?

Reply:
Windows 7.  I suspect that the work that you did may have fixed it, but I haven't checked yet.